yat-docs/docs/partner-guides/1_crypto_wallets.md

```diff
@@ -0,0 +1,304 @@
+        "refresh_token": "eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJiNmI2ZGE4MS0yNTBlLTQ1NmUtYTYxYS01ZjQyNmFlOTJhMjIiLCJpc3MiOiJlbW9qaS1pZC10b2tlbi1pc3N1ZXIiLCJleHAiOjE2MjMwNTQ4MjgsInNjb3BlcyI6WyJ0b2tlbjpyZWZyZXNoIl0sImlzc3VlZCI6MTYyMDM3NjQyOCwiYWN0aXZlMmZhIjowfQ.8Tr8zLbMLNzleG9KL0zJihBI6cd5V8rXPIOjQjcB_WI",
+        "requires_2fa": null
+    },
+    "user": {
+        "id": "b6b6da81-250e-456e-a61a-5f426ae92a22",
+        "alternate_id": "keith24",
+        "is_active": true,
+        "pubkeys": [
+            "ac7f4164608a0b75a3d97da9bd27dc957c2ee80462875daf5e7ba18bf3dce545"
+        ],
+        "free_limit": 0,
+        "remaining_free_emoji": 0
+    },
+}
+```
+
+### Redirection of the User to the Yat Web Application
+
+The partner application user needs to get redirected to the Yat website to Get a Yat (Flow [1A](#1a-get-a-yat-custodial-wallets) & [1B](#1b-get-a-yat-non-custodial-wallets)) or Connect a Yat (Flow [2](#2-connect-existing-yats)). A number of query parameters are necessary for the Yat web application to correctly identify the user and link any necessary data to the user's Yat.
+
+Web App URL: `https://y.at/partner/{partner_path}`
+
+**Path Parameters:**
+
+`partner_path:` A short path parameter for the partner that is defined by the Yat team and delivered to the partner application development team.
+
+**Query Parameters:**
+
+`refresh_token:` User's refresh token that was received in the response body of redirection to Yat web or as a deep link query parameter in redirection from Yat web in both of the above flows - Get a Yat (Flow [1A](#1a-get-a-yat-custodial-wallets) & [1B](#1b-get-a-yat-non-custodial-wallets)) or Connect a Yat (Flow [2A](#2a-connect-existing-yats-from-onboarding) & [2B](#2b-connect-existing-yats-from-settings)).
+
+`eid:` The Yat that the user linked or purchased, received by the partner application as a result of the call to the /process endpoint in Step #2a or as a deep link query parameter in Flow [8](#8-enable-yat-users-to-connect-with-partner-wallets) (Enable Yat users to connect with partner wallets).
+
+`addresses:` A percent-encoded series of cryptocurrency addresses to be linked to the Yat separated by pipes in the format `{YAT_TAG_1}={ADDRESS_1}|{YAT_TAG_2}={ADDRESS_2}|...|{YAT_TAG_N}={ADDRESS_N}`.
+Please refer to the [Yat record categories](https://api-docs.y.at/docs/categories) for detailed information on record categories, but here's a partial list of cryptocurrency address tags:
```

Comment:
@NovaT82  Please update the categories page. You can pull the full list from https://github.com/tari-labs/emoji.id/blob/main/yat_types/src/category.rs#L479